### DISCUSSION TITLE
More Minimalism

### DISCUSSION THREAD

**Discussion by daobrien** [StyleGuides/WritingStyleGuide#625] [General]
This started in Slack but it needs more eyes.

Here's an example of what I'm doing:
(Original:)
Use the following `systemctl status` command to view the status of the `sshd` service. Take note of the process identifier (PID):
• View the status of the `sshd` service.
~~~
[user@host ~]$ systemctl status sshd.service
...output omitted...
~~~
Updated version:
• View the status of the `sshd` service, and take note of the process identifier (PID):
~~~
[user@host ~]$ systemctl status sshd.service
...output omitted...
~~~

I've just saved about a dozen words in one bullet. afaict we haven't lost any info.
Will this work everywhere? No. Sometimes we need to go the whole hog, but I've seen many occasions where this does work, and so far haven't seen any down sides. 

Generally speaking, instead of "Use the `whatever` command to view the `something` status", just write "View the `something` status", and everything else falls into the actual command example.

**Comment by daobrien** [comment]
Opportunities for minimalism are everywhere. Sometimes we have to be a bit careful, but compare these two:

"review the contents of the `/etc/resolv.conf` file"
"review the `/etc/resolv.conf` file"

Yes, there are times when you need to be more specific:
"Review the permissions on the `/etc/resolv.conf` file."

Like most things, always and never don't apply.

**Comment by pagomez-cd** [comment]
I like the suggestion, specially because I think that the information in the original version is repeated in the following code block. The updated version goes to the point.

I know that the discussion is not about this subject, but I think that the "take note" part sounds a bit weird and it barely adds to the exercise. We should be more intentional about the information that we want the student to note. For example:

- View the status of the sshd service. Systemd assigns a new process identifier (PID) for the service:

**Reply by daobrien** [reply]
I also look sideways at that sometimes. Sometimes students do need to take notice of it (for later reference) but sometimes it's just "hey look".
